back-end/src/config/typeorm.config.ts

```diff
@@ -1,17 +1,18 @@
+import { ConfigService } from '@nestjs/config';
 import { TypeOrmModuleOptions } from '@nestjs/typeorm';
 import { Alert } from 'src/alert/entities/alert.entity';
 import { Friend } from 'src/friend/entities/friend.entity';
 import { GameRecord } from 'src/game/entities/game.entity';
 import { User } from 'src/users/entities/user.entity';
 
+const configService = new ConfigService();
+
 export const typeORMConfig: TypeOrmModuleOptions = {
   type: 'postgres',
-  // host: 'database'
-  host: 'localhost',
-  password: 'password',
-  port: 5432,
-  // username: 'transland',
-  username: 'postgres',
+  host: configService.get('DB_HOST'),
+  password: configService.get('DB_PASSWORD'),
+  port: configService.get('DB_PORT'),
+  username: configService.get('DB_USERNAME'),
   database: 'transcendence',
```

Comment:
```suggestion
  database: configService.get('DB_DATABASE'),
```

`database` 도 config에서 불러오는게 좋지 않을까요?

Reply:
https://docs.nestjs.com/techniques/configuration#custom-env-file-path

`.env.dev`, `.env.dev.local` 파일 등 여러 파일에서 먼저 찾은 파일에서만 .env를 읽어오는 기능이 존재합니다.
즉, `.env.dev.local` 파일이 있다면 `.env.dev.local` 파일에서 env를 읽고, 없다면 `.env.dev` 에서 읽어오는 식입니다.

이 기능을 활용하면 도커기준, 로컬기준을 생각할 이유가 없습니다.

또한 **master에 올라가는 코드는 실제 배포 단계에 있다고 간주되는 코드** 라고 저희가 저번 회의에서 이야기 했던 것으로 기억합니다. 따라서 임시처리된 로컬 기준 코드를 master에 올리는 것 보다, 차라리 이러한 방식으로 env를 읽는 것이 맞다고 생각합니다.